nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/summary/nf-summary-table.js

```diff
@@ -417,7 +426,7 @@
                 var markup = '';
 
                 if (isInShell) {
-                    markup += '<div class="pointer go-to fa fa-long-arrow-right" title="Go To Processor" style="margin-right: 3px;"></div>';
+                    markup += '<div class="pointer go-to fa fa-long-arrow-right" title="Go To ' + nfCommon.escapeHtml(dataContext.processGroupNamePath) + ' Processor" style="margin-right: 3px;"></div>';
```

Comment:
I'd suggest:
````Go To Processor in <path>````
instead of:
````Go To <path> Processor````

Or, otherwise, the path should contain the processor name itself but I feel like it would be redundant.

Reply:
I've pushed the change. Thanks @pvillard31.